test-harness/local-run.sh

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env bash
```

Comment:
Thank you for making local testing easier!!

Unsure how close we can get here, but what do you see as the absolute closest we can have our local testing match the testing from the pipeline steps? This is always a huge pain point that I have. If you could solve it (perhaps this approach already does) it would be a huge win IMO.

Reply:
at a minimum, both `local-run` and `azure-pipelines` should use the same test harness container(that's coming soon with multi stage builds). I'd have to think about an effective way to locally replicate the branch naming convention azure devops follows.